tests/storage/test_blob_storage_artifacts.py

```diff
@@ -0,0 +1,81 @@
+#    Copyright 2022 Neal Lathia
```

Comment:
The `blob_storage.py` has become quite big. Instead of re-thinking the whole thing (in this PR), I've split up the test file - which was huge - into smaller ones. This makes it easier to navigate for me